### PR TITLE
Fixes #1613

### LINKED ISSUE #1613: Add sample for showing analytics support for forms
Add a sample similar to https://www.ampproject.org/docs/reference/components/amp-form#analytics-triggers

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

File: src/20_Components/amp-analytics.html [hunk 1]
```diff
@@ -341,6 +346,69 @@
...
+    <div class="ampstart-input inline-block relative m0 p0 mb3">
+      <input type="text"
+          class="block border-none p0 m0"
+          name="name"
+          placeholder="Name..."
+          required>
+    </div>
+    <input type="submit" value="Subscribe" class="ampstart-btn caps">
+    <div submit-success>
+      <template type="amp-mustache">
```
Comment (sebastianbenz) [inline src/20_Components/amp-analytics.html]: I think you can simplify this by removing the templates (then you can also remove the amp-mustache import)

File: src/20_Components/amp-analytics.html [hunk 2]
```diff
@@ -19,12 +20,16 @@
   <title>amp-analytics</title>
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <!-- ## Setup -->
-  <!--  Import the amp-analytics component in the header. -->
+  <!--  Import the `amp-analytics` component in the header. -->
   <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
-  <!--  Import the amp-carousel component in the header. -->
+  <!--  Import the `amp-carousel` component in the header. -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-analytics.html]: I think it's fine to remove the documentation from all non analytics related extensions as I don't think they need to be listed on the doc page. Wdyt?

Comment (kul3r4) [reply src/20_Components/amp-analytics.html]: done

File: src/20_Components/amp-analytics.html (hunk 1 above)
Comment (kul3r4) [reply src/20_Components/amp-analytics.html]: done

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Merging now as I'm going to make another deployment

**Pull request merged**